app/views/talk/show.html.erb

```diff
@@ -1,2 +1,4 @@
-<div><p class="help-text" style="margin-top: 10px;font-family: 'Junction Light';clear:left;"><%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %></p></div>
+<div><p class="help-text" style="margin-top: 10px;clear:left;">
+  <%= raw t('talk.show.welcome', :page => @node.latest.title, :url1 => @node.path, :url2 => '/wiki/talk-pages') %> | for help, <a href="/chat">visit our chatroom</a>
```

Comment:
Just one more change here - could you change the format of `:page => _____` and similar assignments to `page: ______` ? This is the "newer" Ruby syntax, and we're slowly trying to transition. Thank you!!!

Reply:
@jywarren Hi, i'll make those changes, but are these the corrections that address my errors?

Reply:
Sorry, missed this reply -- my bad! No, they're just stylistic things that help long-term maintainability. Let me take a look at your errors!

Reply:
I thought I'd replied to those, my apologies for missing that!